gravitee-am-management-api/gravitee-am-management-api-standalone/gravitee-am-management-api-standalone-distribution/src/main/resources/templates/i18n/messages_en.properties

```diff
@@ -32,6 +32,11 @@ email.reset_password.header.description=If you didn't ask to reset your password
 email.reset_password.button=Reset my password
 email.reset_password.description=This link will expire in {0,number,integer} {1}. After that, you must submit a new request to ask for a new password.
 
+email.verify_attempt.subject=Multiple verification attempt detected
```

Comment:
Same changes requested as per the gateway translation files.

Reply:
changed the other properties file